pkg/components/componenets_config.yaml

```diff
@@ -0,0 +1,25 @@
+componenets:
+  multus:
+    componentName: "multus"
```

Comment:
If we keep components under key with their name, let's drop redundant "componentName".